### PR TITLE
Adding ESQ to our open source projects

### PR DESCRIPTION
**Pull request by jodiedoubleday**
## What does this PR do?
Creates a project page for ESQ 
- An easy way to build elasticsearch queries without having to writing lots and lots of code
Adds that page to the project page on Hungry Geek.

## How to test
- Run ```gem install jekyll```
- Run ```bundle install```
- Run ```jekyll serve``` to load up hungry geek locally.
- Visit http://127.0.0.1:4000/projects/ you should see a link to ESQ
- Visit http://127.0.0.1:4000/projects/esq/

#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this on mobile, tablet and desktop for any visual changes.
- [x] I am ready for this to be code reviewed, merged and tested.

#### IP or Developer Review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

#### Software Engineer, Software Architect or Developer review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

### CODE REVIEW

**Comment by rahulpatel** [comment]
I'll take SE/SA/Dev review...

File: projects/esq.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+layout: project
+title: ESQ (Elasticsearch Query)
+tagline: An easy way to build elasticsearch queries without having to writing lots and lots of code
+date: 2014-12-31 12:00:00
+tags: elastic-search es node npm
```
Comment (rahulpatel) [inline projects/esq.md]: Here are the keywords from `package.json` of esq:

```
    "elasticsearch",
    "esq",
    "querying",
    "queries",
    "query",
    "elastic",
    "search",
    "dsl"
```

**Comment by rahulpatel** [comment]
Just one comment, nothing stopping me from +1'ing though :)

**Comment by stuartstobie** [comment]
Checking this out

**Comment by stuartstobie** [comment]
Looks good, bar the headers being [a bit smushed](http://new.tinygrab.com/1b6e06b07cb19bd5b65fe99a3e699adc97e087eaae.png) around 'Documentation'. Not the fault of the content though so +1.

**Pull request merged**